src/excel_generator/task_builder.py

```diff
@@ -0,0 +1,39 @@
+
+class classproperty(object):
+    def __init__(self, fget):
+        self.fget = fget
+
+    def __get__(self, owner_self, owner_cls):
+        return self.fget(owner_cls)
+
+
+class BaseAnalyticReportSettings:
+    def __init__(self,
+                 sheet_name: str,
+                 header_data: tuple[tuple[str]],
+                 footer_data: tuple[str],
+                 row_count: int):
+        self.sheet_name = sheet_name
+        self.header_data = header_data
+        self.footer_data = footer_data
+        self.row_coun = row_count
+
+
+class TaskAnalyticReportSettings(BaseAnalyticReportSettings):
+    """Конфигурация отчёта для заданий"""
+
+    sheet_name: str = "Задачи"
+    header_data: tuple[str] = (
+        ("Задача", "Кол-во принятых отчётов", "Кол-во отклонённых отчётов", "Кол-во не предоставленных отчётов"),
+    )

```

Comment:
```suggestion
    header_data: tuple[str] =
        ("Задача", "Кол-во принятых отчётов", "Кол-во отклонённых отчётов", "Кол-во не предоставленных отчётов"),
```
Писали уже сто раз, избавиться от лишней вложенности.